src/samplers/probabilistic_sampler.js

```diff
@@ -26,6 +28,14 @@ export default class ProbabilisticSampler implements LegacySamplerV1 {
     this._samplingRate = samplingRate;
   }
 
+  update(samplingRate: number): boolean {
+    if (this._samplingRate == samplingRate) {
```

Comment:
Is there an equality function which takes in an epsilon for float comparisons?

Reply:
not that I know off. I don't think it matters here. If the rate is coming from JSON strategy, then float representation will be the same as long as the string representation is the same.